docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md

```diff
@@ -181,49 +185,79 @@ You can create a `kubeconfig` file for the previously created `yugabyte-platform
+### Configure region and zones
 
-![Add new region](/images/ee/k8s-setup/k8s-az-kubeconfig.png)
+Continue configuring your Kubernetes provider by clicking **Add region** and completing the **Add new region** dialog as follows:
 
 1. Use the **Region** field to select the region.
 
-1. Use the **Zone** field to select a zone label that should match the value of failure domain zone label on the nodes. `topology.kubernetes.io/zone` would place the pods in that zone.
+1. Click **Add Zone** and complete the corresponding portion of the dialog. Notice that there are might be multiple zones.
+
+1. Use the **Zone** field to select a zone label that should match the value of failure domain zone label on the nodes. For example, `topology.kubernetes.io/zone` would place the pods in that zone.
+
+1. Use **Kube Config** to upload the `kubeconfig` file. If this file is available at the provider level, you are not required to supply it.
 
-1. Optionally, use the **Storage Class** field to enter a comma-delimited value. If you do not specify this value, it would default to standard. You need to ensure that this storage class exists in your Kubernetes cluster and takes into account [storage class considerations](../../../install-yugabyte-platform/prepare-environment/kubernetes/#configure-storage-class).
+1. Optionally, use the **Storage Classes** field to enter a comma-delimited value. If you do not specify this value, it would default to standard. You need to ensure that this storage class exists in your Kubernetes cluster and takes into account [storage class considerations](../../../install-yugabyte-platform/prepare-environment/kubernetes/#configure-storage-class).
 
-1. Use the **Namespace** field to specify the namespace. If provided service account has the `Cluster Admin` permissions, you are not required to complete this field. The service account used in the provided `kubeconfig` file should have access to this namespace.
+1. In the **Kube Domain** field, provide the DNS domain name used in the Kubernetes cluster.
 
-1. Use **Kube Config** to upload the configuration file. If this file is available at the provider level, you are not required to supply it. 
+1. Use the **Kube Namespace** field to specify the namespace. If the provided service account has the `Cluster Admin` permissions, you are not required to complete this field. The service account used in the provided `kubeconfig` file should have access to this namespace.
 
 1. Complete the **Overrides** field using one of the provided [options](#overrides). If you do not specify anything, YugabyteDB Anywhere uses defaults specified inside the Helm chart. For additional information, see [Open source Kubernetes](../../../../deploy/kubernetes/single-zone/oss/helm-chart/).
 
-1. Click **Add Zone** and complete the corresponding portion of the dialog. Notice that there are might be multiple zones.
+If required, add a new zone by clicking **Add Zone**, as your configuration may have multiple zones.
```

Comment:
Another field after Overrides is missing that needs to be added- "Cert-Manager Issuer Type"